Задания с 18.02.2022/Task2/Task2/BWT.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+
+public class BWT
+{
+    private static string leftShift(string line, int count)
+    {
+        return line.Substring(count) + line.Substring(0, count);
+    }
+
+    public static string Direct(string line, ref int endPosition)
+    {
+        string[] cyclicShifts = new string[line.Length];
+        for (var i = 0; i < line.Length; i++)
+        {
+            cyclicShifts[i] = line;
+            line = leftShift(line, 1);
+        }
+        Array.Sort(cyclicShifts);
+
+        string result = "";
+        for (var i = 0; i < line.Length; i++)
+        {
+            result += cyclicShifts[i][cyclicShifts[i].Length - 1];
+            if (String.Compare(line, cyclicShifts[i]) == 0)
```

Comment:
```suggestion
            if (line == cyclicShifts[i])
```
В C# есть перегрузка операторов, и `==` честно сравнивает строки (в Java, кстати, нет, там надо строки через Equals сравнивать)